courses/free-courses-en.md

```diff
@@ -516,6 +516,7 @@
 * [Advanced NLP using spaCy](https://course.spacy.io) - Ines Montani
 * [Complete Deep Learning](https://www.youtube.com/playlist?list=PLZoTAELRMXVPGU70ZGsckrMdr0FteeRUi) - Krish Naik
 * [Convolutional Neural Networks for Visual Recognition](http://cs231n.github.io)
+* [Deep Learning Autumn 2018 - Stanford CS 230](https://www.youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb) (Stanford University)
```

Comment:
```suggestion
* [Stanford CS230: Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb) (Stanford University)
```

Reply:
This change was also made, creating a new branch, and a new Pull Request which has been numbered #9846. Please review.